tests/unit/modules/test_dpkg_lowpkg.py

```diff
@@ -20,8 +20,41 @@
 import salt.modules.dpkg_lowpkg as dpkg
 
 
+DPKG_ERROR_MSG = """dpkg-query: package 'httpd' is not installed
+Use dpkg --contents (= dpkg-deb --contents) to list archive files contents.
+"""
+
+DPKG_L_OUTPUT = {"hostname": """\
+/.
+/bin
+/bin/hostname
+/usr
+/usr/share
+/usr/share/doc
+/usr/share/doc/hostname
+/usr/share/doc/hostname/changelog.gz
+/usr/share/doc/hostname/copyright
+/usr/share/man
+/usr/share/man/man1
+/usr/share/man/man1/hostname.1.gz
+/bin/dnsdomainname
+/bin/domainname
+/bin/nisdomainname
+/bin/ypdomainname
+/usr/share/man/man1/dnsdomainname.1.gz
+/usr/share/man/man1/domainname.1.gz
+/usr/share/man/man1/nisdomainname.1.gz
+/usr/share/man/man1/ypdomainname.1.gz
+"""}
+
+
 @skipIf(NO_MOCK, NO_MOCK_REASON)
 class DpkgTestCase(TestCase, LoaderModuleMockMixin):
+    def dpkg_L_side_effect(self, cmd, **kwargs):
+        self.assertEqual(cmd[:2], ["dpkg", "-L"])
+        package = cmd[2]
+        return DPKG_L_OUTPUT[package]
+
     '''
```

Comment:
Looks like the method `dpkg_L_side_effect` should be below the doc string for the test class

Reply:
Done.